tests/store/api/find/test_find_business_data_sample.py

```diff
@@ -85,6 +85,40 @@ def test_get_samples_by_subject_id_and_is_tumour(
     assert samples and len(samples) == 1
 
 
+def test_get_samples_by_customer_id_list_and_subject_id_and_is_tumour(
```

Comment:
Consider writing some additional tests, for instance filtering on a non existing customer id.

Reply:
Sounds good, I have added a unit test for a non-existing customer.